test/__snapshots__/validate.test.ts.snap

```diff
@@ -483,6 +483,66 @@ https://functionless.org/docs/error-codes#eventbus-rules-do-not-support-integrat
 
 [7m39[0m   return (func() as any) === \\"x\\";
 [7m  [0m [91m          ~~~~~~[0m
+[96mtest/test-files/event-bus.ts[0m:[93m43[0m:[93m34[0m - [91merror[0m[90m Functionless(10030): [0mEvent Bridge does not support a truthy comparison in
+
+https://functionless.org/docs/error-codes#event-bridge-does-not-support-a-truthy-comparison-in
+
+[7m43[0m bus.when(\\"rule\\", (event) => <any>event);
```

Comment:
This should be a different error? Maybe its fine for now, now sure what it means for the event not to exist.

Reply:
I just wanted to test when it was an `identifier`, perhaps it doesn't make sense but i as being thorough - truthy is not allowed